timesketch/tsctl.py

```diff
@@ -616,3 +622,91 @@ def validate_context_links_conf(path):
             print(f'=> OK: "{entry}"')
         except (ValidationError, SchemaError) as err:
             print(f'=> ERROR: "{entry}" >> {err}\n')
+
+
+# Analyzer stats cli command
+@cli.command(name="analyzer-stats")
+@click.argument("analyzer_name")
```

Comment:
Since you support an "all" keyword below, maybe this should be mentioned in the help for the command?

Reply:
good point